### ISSUE TITLE
Game crashes on Mac since update

### ISSUE THREAD

**Issue by shadowdive10** [ValveSoftware/Dota2-Gameplay#4920]
### Description

Macbook pro M1 13" (2020)
Game crashes often during gameplay since Battle Pass II update.
I have noticed it happens often during visually intensive abilities (like Monkey Kings Ulti or Zeus' ulti), but also happens during teamfights whilst using any hero basically.

Tried uninstalling and downloading the game again. Tried also setting the graphics on the minimal possible (normally I'd play with the preset 3/4 graphics, 4 being more intensive and 1 being less intensive). None of them worked.

Happens not only during diretide, but also during normal games and even demo mode (demo hero).

### Example Match ID (and possibly Timestamp)

6841529790

**Comment by ivanov-ia** [comment]
Have same issue on Linux with OpenGL.

**Comment by Bahbear** [comment]
Same- fps and ping good, plenty of room on computer…

**Comment by kumel619** [comment]
try "-gl" on lauch options

**Comment by tiritto** [comment]
It might be related to #4851 reported by many Linux players.

**Comment by shadowdive10** [comment]
> try "-gl" on lauch options

Forgot to mention:
Im using OpenGL
I have to force it through launch options by typing this „-gl“ since september or begining of october, since arround this time game ceased to even open with „-vulkan“

**Comment by tiritto** [comment]
> > try "-gl" on lauch options
> 
> Forgot to mention: Im using OpenGL I have to force it through launch options by typing this „-gl“ since september or begining of october, since arround this time game ceased to even open with „-vulkan“

Have you tried using `-vulkan -vulkan_disable_graphics_pipeline_library` launch options?

**Comment by shadowdive10** [comment]
> > > try "-gl" on lauch options
> > 
> > 
> > Forgot to mention: Im using OpenGL I have to force it through launch options by typing this „-gl“ since september or begining of october, since arround this time game ceased to even open with „-vulkan“
> 
> Have you tried using `-vulkan -vulkan_disable_graphics_pipeline_library` launch options?

yes, tried that based on other posts i've seen. Rosetta opens for a fraction of a second, then it closes and dota never really opens :(
